### ISSUE TITLE
How to retrain model with my data！

### ISSUE THREAD

**Issue by fengduanqiao** [xuannianz/EfficientDet#109]
I want to use the model to train my data. Currently I have labeled the data with labelme. What are the specific procedures for the next operation?

**Comment by CraigWang1** [comment]
You can convert it to pascal or coco format

**Comment by PieroCV** [comment]
Hi!
You need to prepare the data with this format:

![image](https://user-images.githubusercontent.com/35885318/77371040-56c79680-6d30-11ea-86e0-cbd32673c10b.png)


this is vest_train.csv

![image](https://user-images.githubusercontent.com/35885318/77371107-870f3500-6d30-11ea-9683-79816b8aeda5.png)

The same format for vest_test.csv

![image](https://user-images.githubusercontent.com/35885318/77371138-98584180-6d30-11ea-9487-c4453dcc995a.png)

Finally, a csv with tags. Here there is only one tag, but you could fill more columns with different tags (<tag>, i = [0,1,2,3...]).

To train, you have to use:
`python train.py --snapshot imagenet --phi 0 --gpu 0 --random-transform --compute-val-loss --freeze-backbone --batch-size 32  csv tags/vest/vest_train.csv tags/vest/vest_names.csv --val-annotations-path tags/vest/vest_val.csv`

**Comment by chenlong8507** [comment]
@PieroCV it works

**Comment by cyx6666** [comment]
> Hi!
> You need to prepare the data with this format:
> 
> ![image](https://user-images.githubusercontent.com/35885318/77371040-56c79680-6d30-11ea-86e0-cbd32673c10b.png)
> 
> this is vest_train.csv
> 
> ![image](https://user-images.githubusercontent.com/35885318/77371107-870f3500-6d30-11ea-9683-79816b8aeda5.png)
> 
> The same format for vest_test.csv
> 
> ![image](https://user-images.githubusercontent.com/35885318/77371138-98584180-6d30-11ea-9487-c4453dcc995a.png)
> 
> Finally, a csv with tags. Here there is only one tag, but you could fill more columns with different tags (, i = [0,1,2,3...]).
> 
> To train, you have to use:
> `python train.py --snapshot imagenet --phi 0 --gpu 0 --random-transform --compute-val-loss --freeze-backbone --batch-size 32 csv tags/vest/vest_train.csv tags/vest/vest_names.csv --val-annotations-path tags/vest/vest_val.csv`

Hi! How to prepare the data for csv format?

**Comment by PieroCV** [comment]
@cyx6666 Using brute force haha...
There are not Scripts to create the csv data, but it's not difficult.
For example, most of labeling software returns a pascal voc format in xml files. There are Scripts to convert this information in csv. Then, take this csv with pandas and modify some columnas to get the right format in order to train a Efficientdet model with this repo.
